utils.go

```diff
@@ -16,20 +17,33 @@ const (
 	TagsHTTPHeaderPrefix = "Open-Tracing-Trace-Tags-"
 )
```

Comment:
Yeah, I'd like to (a) remove this, (b) move `standardtracer` into the `examples` package or similar, and (c) further generalize the utils stuff per your other comments.

In my mind the above could and should be its own PR, though.